File: eSouvenirMobile/app/src/main/java/hr/algebra/esouvenir/model/Picture.kt
package hr.algebra.esouvenir.model

data class Picture (

    var _id: Int?,
    val name: String,
    val contentType: String?,
    val content: ByteArray?,
    val picturePath: String,
    var accomodationID: Int?,
    var walletID: Int?,
    var popularLocationID: Int?

        ) {

    override fun toString(): String {
        return ""
    }
}

File: eSouvenirWebApp/eSouvenirWebApp/Models/Picture.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace eSouvenirWebApp.Models
{
    public class Picture
    {
        public int IDPicture { get; set; }
        public string Name { get; set; }
        public string PicturePath { get; set; }
        public int AccomodationID { get; set; }
        public int PopularLocationID { get; set; }
        public int WalletID { get; set; }
    }
}
